source/devices/J7_Family/linux/Release_Specific_Yocto_layer_Configuration.rst

```diff
@@ -8,23 +8,42 @@ Yocto Layer Configuration
 
 .. http://processors.wiki.ti.com/index.php/Processor_SDK_Building_The_SDK#Layer_Configuration
 
-|__SDK_FULL_NAME__| uses the 'oe-layersetup' tool to configure the meta layers. If you do not have the Linux SDK package installed:
-::
+.. ifconfig:: CONFIG_image_type in ( 'adas', 'edgeai')
 
-    git clone https://git.ti.com/git/arago-project/oe-layersetup.git yocto-build
-    cd yocto-build
-    ./oe-layertool-setup.sh -f configs/processor-sdk-linux/<Config File>
+    |__SDK_FULL_NAME__| uses the 'oe-layersetup' tool to configure the meta layers. If you do not have the Linux SDK package installed:
+    
+    .. code-block:: console
+
+        git clone https://git.ti.com/git/arago-project/oe-layersetup.git yocto-build
+        cd yocto-build
+        ./oe-layertool-setup.sh -f configs/processor-sdk-analytics/<Config File>
+
+.. ifconfig:: CONFIG_image_type in ('default')
+
+    |__SDK_FULL_NAME__| uses the 'oe-layersetup' tool to configure the meta layers. If you do not have the Linux SDK package installed:
+    
+    .. code-block:: console
+
+        git clone https://git.ti.com/git/arago-project/oe-layersetup.git yocto-build
+        cd yocto-build
+        ./oe-layertool-setup.sh -f configs/processor-sdk-linux/<Config File>
 
-The Linux SDK package also has the above tool cloned at the top level. If you have it installed:
```

Comment:
@ravi-rahul @sinha-shreyash 
Was this removed intentionally? Do you want to just keep the one liner :
"The Linux SDK package also has the above tool cloned at the top level of the install location."

Reply:
Added this back